components/test_storage/src/sync_storage.rs

```diff
@@ -154,8 +156,8 @@ impl<E: Engine> SyncTestStorage<E> {
     ) -> Result<Vec<Option<Vec<u8>>>> {
         let mut ids = vec![];
         let requests: Vec<GetRequest> = keys
-            .to_owned()
-            .into_iter()
+            .iter()
+            .copied()
```

Comment:
what's the difference between `copied` and `cloned`?

Reply:
I think clippy prefers `copied` over `cloned`, but sometimes `Copy` is not implemented (for `Vec`).